### PR TITLE
[JSDK-2659]Added updated styling to be more consistent with examples

### PR DESCRIPTION
**Pull request by PikaJoyce**
Redesigned the UI so that it's more consistent with the Examples page. 

With these changes, we would probably need to change the ReadMe image to reflect the UI changes. Use S3?

Here's what it currently looks like
FinallyDone

image

Demonstrating local and remote participant events upon a participant leaving the room 
YES

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: quickstart/src/index.js [hunk 1]
```diff
@@ -104,11 +129,13 @@ $.getJSON('/token', function(data) {
...
-  }).map(function(publication) {
-    return publication.track;
-  });
+  return Array.from(participant.tracks.values())
+    .filter(function(publication) {
+      return publication.track;
+    })
+    .map(function(publication) {
+      return publication.track;
+    });
```
Comment (manjeshbhargav) [inline quickstart/src/index.js]: @PikaJoyce I don't think this styling change is necessary.

File: quickstart/src/index.js [hunk 2]
```diff
@@ -168,16 +197,19 @@ document.getElementById('button-preview').onclick = function() {
...
+    function(tracks) {
+      window.previewTracks = previewTracks = tracks;
+      var previewContainer = document.getElementById('local-media');
+      if (!previewContainer.querySelector('video')) {
+        attachTracks(tracks, previewContainer);
+      }
+    },
+    function(error) {
+      console.error('Unable to access local media', error);
+      log('Unable to access Camera and Microphone');
```
Comment (manjeshbhargav) [inline quickstart/src/index.js]: @PikaJoyce I don't think this styling change is necessary.

File: README.md [hunk 3]
```diff
@@ -1,39 +1,41 @@
...
 
 This application should give you a ready-made starting point for writing your
 own video apps with Twilio Video. Before we begin, we need to collect
 all the config values we need to run the application:
 
-* Account SID: Your primary Twilio account identifier - find this [in the console here](https://www.twilio.com/console).
-* API Key: Used to authenticate - [generate one here](https://www.twilio.com/console/runtime/api-keys).
-* API Secret: Used to authenticate - [just like the above, you'll get one here](https://www.twilio.com/console/runtime/api-keys).
+- Account SID: Your primary Twilio account identifier - find this [in the console here](https://www.twilio.com/console).
+- API Key Sid: Used to authenticate - [generate one here](https://www.twilio.com/console/runtime/api-keys).
```
Comment (manjeshbhargav) [inline README.md]:
```suggestion
- API Key SID: Used to authenticate - [generate one here](https://www.twilio.com/console/runtime/api-keys).
```

File: .env.template [hunk 4]
```diff
@@ -1,3 +1,3 @@
 TWILIO_ACCOUNT_SID=ACxxxxxxxxxx
-TWILIO_API_KEY=SKxxxxxxxxxx
-TWILIO_API_SECRET=xxxxxxxxxx
+TWILIO_API_KEY_SID=SKxxxxxxxxxx
+TWILIO_API_KEY_SECRET=xxxxxxxxxx
```
Comment (manjeshbhargav) [inline .env.template]: @PikaJoyce On second thought, I don't think we should change this because existing users might get affected by this when they pull in the latest changes from `master`. Suddenly their app will no longer work.

File: quickstart/src/index.js (hunk 2 above)
Comment (PikaJoyce) [reply quickstart/src/index.js]: Sorry. This was done by my auto-matter.

File: .env.template (hunk 4 above)
Comment (PikaJoyce) [reply .env.template]: Would this be the case even if I've changed not only the template but all the variables in the files that use them as well?

Comment (timmydoza) [reply .env.template]: I think this change should still be made in the Ahoy app. There are no existing users that would be affected. 

https://github.com/twilio/twilio-video-app-react/pull/75

This change would mean that the App and the Quickstart would have different names for their environment variables. Think that is ok?

Comment (PikaJoyce) [reply .env.template]: I think it's okay. But maybe need thoughts from @manjeshbhargav. I'm not sure if it's more important to keep it consistent across all platforms, and simply notifying folks of this change and having everyone change their personal .env file OR just changing the read me so it makes more sense intuitively?

File: .env.template [hunk 5]
```diff
@@ -1,3 +1,3 @@
 TWILIO_ACCOUNT_SID=ACxxxxxxxxxx
-TWILIO_API_KEY=SKxxxxxxxxxx
+TWILIO_API_SID=SKxxxxxxxxxx
```
Comment (manjeshbhargav) [inline .env.template]: @PikaJoyce I think we should revert this change as well.

File: .env.template (hunk 4 above)
Comment (manjeshbhargav) [reply .env.template]: @PikaJoyce @timmydoza I think the differences in names are fine, since we don't want to break existing QuickStart users, and the Ahoy App is new. However, the description for how to create these strings should be the same.
